drivers/espi/espi_mchp_xec.c

```diff
@@ -770,6 +769,43 @@ static void notify_system_state(struct device *dev,
 	espi_send_callbacks(&data->callbacks, dev, evt);
 }
 
+static void notify_host_warning(struct device *dev,
+				enum espi_vwire_signal signal)
+{
+#ifndef CONFIG_ESPI_AUTOMATIC_WARNING_ACKNOWLEDGE
+	struct espi_xec_data *data = (struct espi_xec_data *)(dev->driver_data);
+	struct espi_event evt = { ESPI_BUS_EVENT_VWIRE_RECEIVED, 0, 0 };
+#endif
+	u8_t status;
+
+	espi_xec_receive_vwire(dev, signal, &status);
+
+#ifndef CONFIG_ESPI_AUTOMATIC_WARNING_ACKNOWLEDGE
+	evt.evt_details = signal;
+	evt.evt_data = status;
+	espi_send_callbacks(&data->callbacks, dev, evt);
+#else
```

Comment:
Seems like this could use use `if (!IS_ENABLED(CONFIG_ESPI_...)) {` instead? That way you don't need two separate conditional sections and can put the variable declarations inside the branch.

Reply:
Done